### PR TITLE
Allow cancel upgrade prompt

### LINKED ISSUE #1514: Allow cancel upgrade prompt
##### Are there any related issues?
#913 

...

##### What operating system are you using?
ubuntu 18.04.2
...

##### What version of Mailspring are you using?
 1.6.0-10f5c95
...

--

**Bug?**

##### Is this an email sync or connection issue?

...

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

...

##### Is the issue reproducible with a particular attachment, message, signature, etc?

...

--

**Feature Request?**

##### Does this feature exist in another mail client or tool you use?
User can cancel the upgrade notification "Download now" ,Because some new version has bug, and I don't want to upgrade.
image

...

### PR DESCRIPTION
**Pull request by mhatvan**
Fixes #1514, #913.

### CODE REVIEW

**Comment by mhatvan** [comment]
@bengotow the appveyor build fails with openssl errors,i don't think these errors are related to my PR.

Apart from that, I encountered issues with `UndecoratedSendActionButton` import in `send-action-button-spec.jsx` and `'getRemoteConfig is not a function'` in `update-notification-spec.jsx`, which occur when running `yarn test`, even in master branch. 

Therefore this PR is only in draft state until the issues are resolved.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,21 +24,21 @@ You'll need git and a recent version of Node.JS (any v7.2.1+ is recommended with
...
-* make
-* A proper C/C++11 compiler tool chain, for example GCC
-* Library dependencies:
-  * On Debian-based Linux: `sudo apt-get install libx11-dev libxkbfile-dev execstack libgconf2-4 libsecret-1-dev`
-  * On Red Hat-based Linux: `sudo yum install libX11-devel.x86_64 libxkbfile-devel.x86_64 libsecret-1-dev libsecret-devel`.
+- Python v2.7
+- make
+- A proper C/C++11 compiler tool chain, for example GCC
+- Library dependencies:
+  - On Debian-based Linux: `sudo apt-get install libx11-dev libxkbfile-dev execstack libgconf-2-4 libsecret-1-dev`
```
Comment (bengotow) [inline CONTRIBUTING.md:40]: Ahh good catch - thanks for fixing this!

File: app/internal_packages/notifications/specs/update-notification-spec.tsx [hunk 2]
```diff
@@ -81,6 +81,14 @@ describe('UpdateNotification', function describeBlock() {
         notif.find('#action-0').simulate('click'); // Expects the first action to be the install action
         expect(ipcSendArgs).toEqual(['command', 'application:install-update']);
       });
+
+      it('should dismiss the update notification prompt', () => {
```
Comment (bengotow) [inline app/internal_packages/notifications/specs/update-notification-spec.tsx:85]: Thanks for contributing a test! I'm going to rebase this on master and I'll see why those other test failures are happening.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mcmxcdev) [reply CONTRIBUTING.md:40]: had to install it myself on ubuntu, that's how i came across it ;)

**Pull request merged**